addons/point_of_sale/static/src/app/store/product_configurator_popup/product_configurator_popup.xml

```diff
@@ -143,7 +143,7 @@
                         class="btn btn-primary btn-lg lh-lg o-default-button"
                         t-att-class="{'disabled': isArchivedCombination()}"
                         t-on-click="confirm">
-                        Ok
+                        Add
```

Comment:
Don't forget to add this in the .po file

Reply:
done :)